06_neighborhoods/Transportation/transit_cost_county.R

```diff
@@ -0,0 +1,189 @@
+# the Housing and Transportation (H+T) Affordability Index from the Center for Neighborhood Technology (CNT)
+
+###################################################################
+
+# 1. Import all the files (and/or combine into one file) with only the relevant variables and years
+
+# FOR COUNTIES - import the raw data files
+# Unlike for Places and Tracts, the CNT website allows you to download raw data for all counties at once
+# so we just import the all-county file for the two available years; 2015 and 2019
+# 2015
+# raw data lives in the Urban Box folder (replace USERNAME)
+transport_county_2015 <- read_csv("C:/Users/USERNAME/Box/Lab/Projects/Gates Upward Mobility Framework/Outreach and Tools/Data/Metrics_2024_round/Transportation/htaindex2015_data_counties.csv")
+
+# create correct FIPS columns
+transport_county_2015 <- transport_county_2015 %>%
+  mutate(
+    state = substr(county, start = 2, stop = 3),
+    county = substr(county, start = 4, stop = 6)
+  )
+
+# keep only variables of interest
+transportation_cost_county_2015 <- transport_county_2015 %>%
+  select(state, county, blkgrps, population, households, t_80ami)
+
+
+
+#2019
+transport_county_2019 <- read_csv("C:/Users/USERNAME/Box/Lab/Projects/Gates Upward Mobility Framework/Outreach and Tools/Data/Metrics_2024_round/Transportation/htaindex2019_data_counties.csv")
+transport_county_2019 <- transport_county_2019 %>%
+  select(county, blkgrps, population, households, t_80ami)
+
+# create correct FIPS columns
+transport_county_2019 <- transport_county_2019 %>%
+  mutate(
+    state = substr(county, start = 2, stop = 3),
+    county = substr(county, start = 4, stop = 6)
+  )
+
+# keep only variables of interest
+transportation_cost_county_2019 <- transport_county_2019 %>%
+  select(state, county, blkgrps, population, households, t_80ami)
+
+
+# Compare to our official county file to make sure we have all counties accounted for
+# Bring in the official county file
+# you should already be in the project folder for this path to work
+counties <- read_csv("geographic-crosswalks/data/county-populations.csv")
+counties_2015 <- counties %>%
+  filter(year == 2015)
+counties_2019 <- counties %>%
+  filter(year == 2019)
+# all files have same number of observations (3142) so no merging needed to account for missings!
+
+###################################################################
+
+# 2. QC Checks
+  # create a histogram plot and summary stats for each dataframe
+  # check that all values are non-negative & count missing values
+  # examine outliers
+  
+
+  # County-Level Transportation Cost 2015
+ggplot(transportation_cost_county_2015, aes(x=t_80ami)) + geom_histogram(binwidth=10) + labs(y="number of counties", x="Annual Transit Cost for the Regional Moderate Income Household, 2015")
+# look at summary stats
+summary(transportation_cost_county_2015$t_80ami)  
+# examine outliers
+transportation_cost_county_2015_outliers <- transportation_cost_county_2015 %>% 
```

Comment:
How did you decide on 100 for outliers? Not saying it's wrong just not sure what it is based on (a percentile?)